test/Tester/MembershipTests/SilosStopTests.cs

```diff
@@ -50,26 +50,5 @@ public override TestCluster CreateTestCluster()
                 Assert.Equal(typeof(SiloUnavailableException), ex.GetBaseException().GetType());
             }
         }
-
-        [Fact, TestCategory("Functional"), TestCategory("Liveness")]
-        public async Task SiloUngracefulShutdown_ClientOutstandingRequestsBreak()
```

Comment:
Ohh, I see, because of SiloUnavailableException.
As we said above, lets put the SiloUnavailableException back and handle it separately.